todo_app/android/.gitignore

```diff
@@ -0,0 +1,13 @@
+gradle-wrapper.jar
+/.gradle
+/captures/
```

Comment:
A repository-ban egy gitignore file-t tárolunk csak, ami kint van a mappaszerkezet root-jában. Innen olvassa fel a git. Amik itt vannak külön gitignore fileok az android, linux, macos, ios alatt azt kérlek töröljük és a benne lévő hivatkozásokat helyezzük el a külső gitignore fileba. Ez segíti az átláthatóságot, csökkenti a duplikációt és könnyebben kezelhető lesz, hogy éppen milyen szabály miatt van ignorálva az adott file. 
Figyelj arra, hogyha feljebb teszed a gitignore hivatkozásokat, akkor előfordulhat, hogy le kell követni útvonalváltoztatással is.